roles/KubevirtCommonTemplatesBundle/templates/kubevirt-base-image-viewrolebinding.yaml.j2

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: golden-image-kubevirt.io:view
+  namespace: "{{ base_images_namespace }}"
+subjects:
+- kind: Group 
+  name: system:authenticated
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: Role
+  name: golden-image-kubevirt.io:view
```

Comment:
this looks like a domain name, but we don't own it. `golden-image.kubevirt.io` would be safer.